README.md

```diff
@@ -1,53 +1,53 @@
-[![Chat](https://img.shields.io/badge/chat-chat%2eholochain%2enet-blue.svg?style=flat-square)](https://chat.holochain.org)
-
+[![Project](https://img.shields.io/badge/Project-Holochain-blue.svg?style=flat-square)](http://holochain.org/)
+[![Forum](https://img.shields.io/badge/Forum-forum%2eholochain%2enet-blue.svg?style=flat-square)](https://forum.holochain.org)
+[![License: CAL 1.0](https://img.shields.io/badge/License-CAL%201.0-blue.svg)](https://github.com/holochain/cryptographic-autonomy-license)
+![Test](https://github.com/holochain/holochain-client-js/actions/workflows/test.yml/badge.svg?branch=main)
 [![Twitter Follow](https://img.shields.io/twitter/follow/holochain.svg?style=social&label=Follow)](https://twitter.com/holochain)
-License: [![License: CAL 1.0](https://img.shields.io/badge/License-CAL%201.0-blue.svg)](https://github.com/holochain/cryptographic-autonomy-license)
 
-This repository contains the core Holochain libraries and binaries.
+# Holochain
 
-This is the most recent and well maintained version of Holochain with a refactored state model (you may see references to it as Holochain RSM).
+Holochain is an open-source framework to develop peer-to-peer applications with high levels of security, reliability, and performance.
+This repository contains the Holochain core libraries.
 
 ## Code Status
 
-This code is in alpha. It is not for production use. The code is guaranteed NOT secure.
+This code is in alpha. It is not for production use. We will be frequently and heavily restructuring code APIs and data chains until Beta.
 
-We will be frequently and heavily restructuring code APIs and data chains until Beta.
+**We are supporting Linux and macOS at this time**. You may or may not be able to build and run Holochain on Windows.
+We will be supporting it in the future.
 
-**We are currently only supporting Linux at this time**. You may or may not be able to successfully build and run Holochain on macOS or Windows. We will definitely be rolling out support for these OSes in the future, but in the meantime please use Linux for development!
+## Holochain Development and Usage
 
-## Making the Holochain binaries available in your shell
-There are a number of contexts and purposes you might be running in which yield different ways to access binaries.
+### Running Holochain applications (hApps)
 
+Holochain apps can be installed and run using the [Holochain Launcher](https://github.com/holochain/launcher). Refer to the Github repository
+for instructions on how to obtain and use it.
 
-### Using nix-shell on a local clone
-Assuming you have [installed the nix shell](https://nixos.wiki/wiki/Nix_Installation_Guide):
+### Developing Holochain applications (hApps)
 
-```
-nix-shell --argstr flavor happDev
-```
+Looking to write a hApp yourself? You need to set up your development environment and then can scaffold your hApp. Refer to the
```

Comment:
according to personal preference of thinking in possibilities rather than necessities:

```suggestion
Looking to write a hApp yourself? You can start by setting up your development environment and then can scaffold your hApp. Refer to the
```

Reply:
Without the dev env you can't develop anything. But sure...